chcemvediet/apps/anonymization/anonymization_pdf.py

```diff
@@ -0,0 +1,87 @@
+import os
```

Comment:
Je to dobre riesenie vytvorit novy subor, alebo to dam vsetko do jednoho `anonymization.py`?

Reply:
@viliambalaz Asi je to lepsie mat oddelene. Akurat funkcie pls nazvi `anonymize_attachment_to_pdf` a `anonymize_attachment_to_odt`. Nazvy suborov mozes nechat.